metis/db/migrations/022_add_indices.rb

```diff
@@ -0,0 +1,20 @@
+Sequel.migration do
+    change do
+      alter_table(:files) do
+        add_index :bucket_id
+        add_index :folder_id
+        add_index :data_block_id
+        add_index :project_name
```

Comment:
Hold on -- is this one worth it though?  I feel that project_name is dense and not sparse.  I believe there will be a performance drop from indexing this on every write with very little gain on read performance.  Same applies to all project_name entries.

Reply:
Yeah I'm probably being overzealous indexing this, I'm not sure we actually even query files by project_name alone at any point.